userappwithbroker/src/main/java/com/microsoft/aad/adal/example/userappwithbroker/Constants.java

```diff
@@ -67,15 +59,7 @@ public String getText() {
 
     enum ClientId {
         ADFSV4("4b0db8c2-9f26-4417-8bde-3f0e3656f8e0"),
-        MOONCAKE("920cd637-4ca3-496b-8a6a-f7c6ca1b2b82"),
-        ONEDRIVE("af124e86-4e96-495a-b70a-90f90ab96707"),
-        OFFICE("d3590ed6-52b3-4102-aeff-aad2292ab01c"),
-        APPCHECK2_BF("f5d01c1c-abe6-4207-ae2d-5bc9af251724"),
-        GUESTCLIENT("ea5c8087-2476-489c-ae03-ad44a2ac399d"),
-        ADFSV3("68a10fc3-ead9-41b8-ac5e-5b78af044736"),
-        ADFSV2("e3fbd64e-3bdc-4fe9-b531-31660912b944"),
-        PING("6b748729-d940-4482-8724-5eb87a817a10"),
-        SHIBBOLETH("d518483c-c15b-4a00-9f59-cff3ffc1077b");
```

Comment:
Why are these removed?

Reply:
I'm not seeing them being used in any test cases.

Reply:
I know we try to use the `OFFICE` client id for testing some 1st party scenarios. I'm not inclined to removing these as we may end up needing them in the future.

Reply:
Added back.